### PR TITLE
Added 4.0. Updated 3.6. Deleted 3.2, 3.4

### PR DESCRIPTION
**Pull request by ovk6**
Added 4.0. Updated 3.6. Deleted 3.2, 3.4
Added docker-entrypoint.sh to template folder.

### CODE REVIEW

File: versions.yaml [hunk 1]
```diff
@@ -40,35 +52,7 @@ versions:
...
-    packages:
-      gosu: *gosu
-      mongodb:
-        gpg: *gpg34
-        major: '3.4'
-        version: '3.4.17'
-  - dir: '3/debian8/3.2'
-    repo: mongodb3
-    tags:
-      - '3.2.21-debian8'
```
Comment (wgrzelak) [inline versions.yaml:61]: Please mark tags `3.2` and `3.4` as **deprecated** first.

**Review by wgrzelak — CHANGES_REQUESTED** [review]

**Comment by wgrzelak** [comment]
Have you moved `docker-entrypoint.sh` to the templates directory, right? Could you include such information in the PR's description here and in the future PRs, please?

**Comment by ovk6** [comment]
@wgrzelak Right, it was moved

File: versions.yaml (hunk 1 above)
Comment (ovk6) [reply versions.yaml:61]: Marked already

File: versions.yaml [hunk 2]
```diff
@@ -15,22 +15,34 @@
...
+  gpg40: &gpg40 9DA31620334BD75D9DCB49F368818C72E52529D4
 
 versions:
+  - dir: '4/debian9/4.0'
+    repo: mongodb4
+    tags:
+      - '4.0.9-debian9'
+      - '4.0-debian9'
+      - '4.0.9'
+      - '4.0'
```
Comment (wgrzelak) [inline versions.yaml:31]: Please also add `latest` tag.

**Review by wgrzelak — CHANGES_REQUESTED** [review]

File: versions.yaml (hunk 2 above)
Comment (ovk6) [reply versions.yaml:31]: Done

**Review by wgrzelak — APPROVED** [review]

**Pull request merged**